views/solution.erb

```diff
@@ -0,0 +1,5 @@
+<h1> You <%=@Winner%>!! </h1>
+<br>
+<form action='/restart' method='get'>
+  <input type= 'submit' value='Play again'>
```

Comment:
I like that you included a restart option for the user even though it wasn't outright stated in the requirements - shows you thought about the user experience :)